test/dotnet-test.Tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+using Xunit;
+
+// Don't let the tests execute concurrently
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
```

Comment:
Seeing some issues running tests on the CI that I don't see locally:

```
D:\j\workspace\debug_windows---54dcb9ec\artifacts\win81-x86\stage2\dotnet.exe compile-csc @D:\j\workspace\debug_windows---54dcb9ec\test\buildBasePathTest\src\Microsoft.DotNet.ProjectModel\obj\Debug\netstandard1.5\dotnet-compile.rsp returned Exit Code 1
D:\j\workspace\debug_windows---54dcb9ec\src\Microsoft.DotNet.ProjectModel\Compilation\LibraryExportBuilder.cs(17,37): warning CS0649: Field 'LibraryExportBuilder._compilationAssets' is never assigned to, and will always have its default value null
D:\j\workspace\debug_windows---54dcb9ec\src\Microsoft.DotNet.ProjectModel\error CS2012: Cannot open 'D:\j\workspace\debug_windows---54dcb9ec\test\buildBasePathTest\src\Microsoft.DotNet.ProjectModel\bin\Debug\netstandard1.5\Microsoft.DotNet.ProjectModel.dll' for writing -- 'The process cannot access the file 'D:\j\workspace\debug_windows---54dcb9ec\test\buildBasePathTest\src\Microsoft.DotNet.ProjectModel\bin\Debug\netstandard1.5\Microsoft.DotNet.ProjectModel.dll' because it is being used by another process.'
Compilation failed.
    1 Warning(s)
    1 Error(s)
```

Trying out this to see if it's parallel test execution that's causing the file contention.